api/src/main/java/com/thesurvey/api/controller/SurveyController.java

```diff
@@ -78,9 +80,9 @@ public ResponseEntity<SurveyResponseDto> getSurvey(
         @ApiResponse(code = 500, message = "서버 내부 오류")
     })
     @PostMapping
-    public ResponseEntity<SurveyResponseDto> createSurvey(Authentication authentication,
-        @RequestBody SurveyRequestDto surveyRequestDto) {
-        return ResponseEntity.ok(surveyService.createSurvey(authentication, surveyRequestDto));
+    public ResponseEntity<?> createSurvey(Authentication authentication,
```

Comment:
Likewise, as we've already specified response type(DTO), this should be specified.